restaking_core/src/ncn.rs

```diff
@@ -33,8 +33,14 @@ pub struct Ncn {
     /// The delegate admin of the NCN
     pub delegate_admin: Pubkey,
 
-    // Reserved space
-    reserved_1: [u8; 32],
+    /// ( For future use ) Authority to update the ncn's metadata
+    pub metadata_admin: Pubkey,
+
+    /// The weight table admin of the NCN
+    pub weight_table_admin: Pubkey,
```

Comment:
let's remove this - will be diff program. can make choice on which thing is singer later and/or add

Reply:
keeping for now